docker-compose.yml

```diff
@@ -12,6 +10,7 @@ services:
       - backend
     volumes:
       - ./frontend:/app
+      - /app/node_modules
```

Comment:
I believe there was a reason why this line was omitted. We want the node_modiles to be built during setup and not be pre installed. Please delete this line.